animations/spells/1st/bless.json

```diff
@@ -0,0 +1,41 @@
+{
+	"effect:aura-bless": [
+		{
+			"trigger": "effect",
+			"preset": "onToken",
+			"file": "jb2a.magic_signs.circle.02.transmutation.complete.yellow",
+			"options": {
+				"size": {
+					"value": 7,
+					"gridUnits": true
+				},
+				"belowTokens": true
+			},
+			"contents": [
+				{
+					"default": true,
+					"predicate": [
+						"settings:persistent"
+					],
+					"options": {
+						"persist": {
+								"value": true,
+								"persistTokenPrototype": true
+							},
+						"tieToDocuments": true,
+						"belowTokens": true
```

Comment:
You keep repeating belowTokens multiple times. Also you have two defaults. Lastly, the non-persistent version will trigger only once. See other effects for examples on how to format persistent vs. non-persistent effects and copy that.